sdk/core/System.ClientModel/src/Convenience/ApiKeyCredential.cs

```diff
@@ -24,14 +29,33 @@ public class ApiKeyCredential
     public ApiKeyCredential(string key) => Update(key);
 #pragma warning restore CS8618 // Non-nullable field is uninitialized. Consider declaring as nullable.
 
-    // Note: this is uses a method instead of a Key property so if an instance of
-    // the type were passed to a serializer that serializes public properties
-    // it doesn't inadvertently expose a secret.  We call it Deconstruct to allow
-    // consistent naming across credential types.  For example, for
-    // NamedKeyCredential, we expect to have a Deconstruct method with two out
-    // params to enable atomicity in reading multiple credential values.
+    // This type uses a method instead of a Key property so that, if an instance
+    // of the type were passed to a serializer that serializes public properties
+    // by default, it doesn't inadvertently expose a secret. The method has the
+    // name Deconstruct to allow consistent naming across credential types.  For
+    // example, a corresponding NamedKeyCredential type will have a Deconstruct
+    // method with two out params to make reading multiple credential values an
+    // atomic operation.
+
+    /// <summary>
```

Comment:
An inline example may be helpful here, as deconstruction isn't yet a well-known concept.  Then again, we don't expect callers to be reading keys, so maybe we don't need it.